meteor-app/imports/ui/pages/workspace/component.js

```diff
@@ -60,23 +63,32 @@ export default class WorkspacePage extends React.Component {
     }
   }
 
-  _rangeFilterOnChange (event) {
+  _rangeFilterOnChange (value) {
+    if (Date.now() - lastFilterChange < 50) {
```

Comment:
https://egghead.io/lessons/javascript-redux-store-methods-getstate-dispatch-and-subscribe